Documentation/PageTsconfig/Templates.rst

```diff
@@ -0,0 +1,128 @@
+.. include:: /Includes.rst.txt
+
+.. index::
+   Page TSconfig; templates
+   templates
+.. _pagetemplates:
+
+
+=========
+templates
+=========
+
+.. versionadded:: 12.0
+
+All Fluid templates rendered by backend controllers can be overridden with own
+templates on a per-file basis. The feature is available for basically all core
+backend modules, as well as the backend main frame templates. Exceptions are
+email templates and templates of the install tool.
+
+.. caution::
+
+   While this feature is powerful and allows overriding nearly any backend
+   template, **it should be used with care**: Fluid templates of the core
```

Comment:
```suggestion
   template, **it should be used with care**: Fluid templates of the Core
```
See https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/GeneralConventions/Glossary.html